annotations.md

```diff
@@ -49,6 +49,7 @@ This specification defines the following annotation keys, intended for but not l
   * This SHOULD be the immediate image sharing zero-indexed layers with the image, such as from a Dockerfile `FROM` statement.
   * This SHOULD NOT reference any other images used to generate the contents of the image (e.g., multi-stage Dockerfile builds).
   * If the `image.base.name` annotation is specified, the `image.base.digest` annotation SHOULD be the digest of the manifest referenced by the `image.ref.name` annotation.
+* **org.opencontainers.image.toc.digest** Digest of the TOC JSON of [eStargz](estargz.md) layer.
```

Comment:
The concept of digest of TOC may be helpful for other usage cases, so is it OK to relax to support not only estargz?

Reply:
Are there other formats which use TOC?

Reply:
Thanks for the suggestion. But if the "filesystem metadata" format is different from the TOC, shouldn't we have a separated key? Or, in following-up PRs, we can define "filesystem metadata" you mention then relax the scope of this key.

Reply:
If the annotation is specific to eStargz, should the key be `org.opencontainers.image.estargz.digest`?

Reply:
@sudo-bmitch Thank you for the suggestion. Modified the key to `org.opencontainers.image.estargz.toc.digest`. I left `toc` here for futural extensibility (e.g. we may want other toc-related annotations in the future like `estargz.toc.size` or something)